--- thunderbird-78.2.2/comm/mail/base/content/specialTabs.js.orig	2020-09-14 01:07:47.922347748 +0200
+++ thunderbird-78.2.2/comm/mail/base/content/specialTabs.js	2020-09-14 01:07:51.568995951 +0200
@@ -1048,7 +1048,7 @@
       Services.prefs.setCharPref("mailnews.start_page_override.mstone", mstone);
     }
 
-    let update = Cc["@mozilla.org/updates/update-manager;1"].getService(
+    let update = AppConstants.MOZ_UPDATER && Cc["@mozilla.org/updates/update-manager;1"].getService(
       Ci.nsIUpdateManager
     ).activeUpdate;
 
