--- thunderbird-115.11.0/gfx/qcms/src/lib.rs.orig	2024-05-13 17:06:54.000000000 +0200
+++ thunderbird-115.11.0/gfx/qcms/src/lib.rs	2024-05-16 22:59:31.685274047 +0200
@@ -5,9 +5,6 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
-// These are needed for the neon SIMD code and can be removed once the MSRV supports the
-// instrinsics we use
-#![cfg_attr(feature = "neon", feature(stdsimd))]
 #![cfg_attr(
     feature = "neon",
     feature(arm_target_feature, raw_ref_op)
